libs/ui/src/lib/account-balances/account-balances.component.html

```diff
@@ -58,3 +58,48 @@
   <tr *matHeaderRowDef="displayedColumns" mat-header-row></tr>
   <tr *matRowDef="let row; columns: displayedColumns" mat-row></tr>
 </table>
+
+<form
+  class="gf-form"
+  [formGroup]="accountBalanceForm"
+  (ngSubmit)="onSubmitAccountBalance()"
+>
+  <table class="gf-table w-100">
+    <tr>
+      <td>
+        <mat-form-field class="px-2 mb-3">
+          <input
```

Comment:
Can you use `mat-datepicker`?

See here for an example: https://github.com/ghostfolio/ghostfolio/blob/main/apps/client/src/app/pages/portfolio/activities/create-or-update-activity-dialog/create-or-update-activity-dialog.html#L159